packages/sanity/src/_internal/cli/commands/backup/enableBackupCommand.ts

```diff
@@ -0,0 +1,48 @@
+import {CliCommandDefinition} from '@sanity/cli'
+import resolveApiClient from '../../actions/backup/resolveApiClient'
+import {defaultApiVersion} from './backupGroup'
+
+const helpText = `
+Examples
+  sanity backup enable DATASET_NAME
+`
+
+const enableDatasetBackupCommand: CliCommandDefinition = {
+  name: 'enable',
+  group: 'backup',
+  signature: '[DATASET_NAME]',
+  description: 'Enable backup for a dataset.',
+  helpText,
+  action: async (args, context) => {
+    const {output, chalk} = context
+    const [dataset] = args.argsWithoutOptions
+    const {projectId, datasetName, token, client} = await resolveApiClient(
+      context,
+      dataset,
+      defaultApiVersion,
+    )
+
+    try {
+      await client.request({
+        method: 'PUT',
+        headers: {Authorization: `Bearer ${token}`},
+        uri: `/projects/${projectId}/datasets/${datasetName}/settings/backups`,
+        body: {
+          enabled: true,
+        },
+      })
+
+      output.print(`${chalk.green(`Enabled daily backups for dataset ${datasetName}.\n`)}`)
```

Comment:
I'm not sure about including the "daily" part here in case we change the schedule for some reason. Could we just remove that word?

Reply:
Removed it.